src/pages/workspace/WorkspaceSettingsCurrencyPage.js

```diff
@@ -0,0 +1,106 @@
+import React, {useState, useMemo, useCallback} from 'react';
+import _ from 'underscore';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import useLocalize from '../../hooks/useLocalize';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithBackButton from '../../components/HeaderWithBackButton';
+import SelectionList from '../../components/SelectionList';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import compose from '../../libs/compose';
+import ONYXKEYS from '../../ONYXKEYS';
+import withPolicy, {policyDefaultProps, policyPropTypes} from './withPolicy';
+import * as Policy from '../../libs/actions/Policy';
+
+const propTypes = {
+    /** Constant, list of available currencies */
+    currencyList: PropTypes.objectOf(
+        PropTypes.shape({
+            /** Symbol of the currency */
+            symbol: PropTypes.string,
+        }),
+    ),
+    ...policyPropTypes,
+};
+
+const defaultProps = {
+    currencyList: {},
+    ...policyDefaultProps,
+};
+
+function WorkspaceSettingsCurrencyPage({currencyList, policy}) {
+    const {translate} = useLocalize();
+    const [searchText, setSearchText] = useState('');
+
+    const getDisplayText = useCallback((currencyCode, currencySymbol) => `${currencyCode} - ${currencySymbol}`, []);
```

Comment:
This callback could be outside of a component, because it does not use any dependencies. This way we can save a little big of memory. However, I am not sure that we can do in the project.

Reply:
Exactly, I thought the same, but did not find something like `WorkspaceUtils` to place it

Reply:
I would say you can put it in the same file with the component, just before. If you don't like the idea, just forget.

Reply:
I'll do that, I actually prefer it, and since more people mentioned, seems like a good idea